### PR TITLE
added feature branch repo managment with the jfrog cli example - blog…

### PR DESCRIPTION
**Pull request by talitz**
… post by tal yitzhak

### CODE REVIEW

File: feature-branch-repo-management-with-cli-example/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Automate Repository Management with the JFrog CLI
```
Comment (eyalbe4) [inline feature-branch-repo-management-with-cli-example/README.md:1]: Let's change the title to *Repo Management GitHub Actions Example*.

File: feature-branch-repo-management-with-cli-example/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
```
Comment (eyalbe4) [inline feature-branch-repo-management-with-cli-example/README.md:2]: I suggest adding a very brief introduction paragraph, describing this example. See my suggestion below. Let's also make sure that the blog indeed includes all the info needed to make this example work.

This example demonstrates how you can use GitHub Actions to automate the creation and deletion of Artifactory repositories, which correspond to feature branches that are created on GitHub.
The full description of how to set up and run this example are available on ---this blog post---.

File: feature-branch-repo-management-with-cli-example/README.md (hunk 1 above)
Comment (talitz) [reply feature-branch-repo-management-with-cli-example/README.md]: Done.

File: feature-branch-repo-management-with-cli-example/README.md (hunk 2 above)
Comment (talitz) [reply feature-branch-repo-management-with-cli-example/README.md]: Done. I'll add the link for the blog post after it will be published.

**Comment by eyalbe4** [comment]
Thanks @talitz! Looks good.
We'll merge this after you commit the blog URL.

**Pull request merged**